### ISSUE TITLE
Flutter Connectivity Compilation Error

### ISSUE THREAD

**Issue by aydin-faruk** [flutter/flutter#51112]
I cannot debug or release to my app when I add Connectivity to my app's pubspec.yaml

![c1](https://user-images.githubusercontent.com/29239513/74929918-94619880-53ed-11ea-9dda-18a882067c7d.PNG)
![c2](https://user-images.githubusercontent.com/29239513/74929931-99bee300-53ed-11ea-82e6-12bd2d7cb16b.PNG)
![c3](https://user-images.githubusercontent.com/29239513/74929953-a3484b00-53ed-11ea-8f59-bfe3a6bbffde.PNG)

Flutter Doctor 
```
[flutter] flutter doctor -v
[√] Flutter (Channel stable, v1.12.13+hotfix.8, on Microsoft Windows [Version 10.0.18363.657], locale tr-TR)
    • Flutter version 1.12.13+hotfix.8 at C:\flutter
    • Framework revision 0b8abb4724 (9 days ago), 2020-02-11 11:44:36 -0800
    • Engine revision e1e6ced81d
    • Dart version 2.7.0

[!] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at C:\Users\Faruk Aydin\AppData\Local\Android\sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: C:\Program Files\Java\jdk-13.0.2\bin\java
    • Java version Java(TM) SE Runtime Environment (build 13.0.2+8)
    X Android license status unknown.
      Try re-installing or updating your Android SDK Manager.
      See https://developer.android.com/studio/#downloads or visit https://flutter.dev/setup/#android-setup for detailed instructions.

[!] Android Studio (version 3.5)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 42.1.1
    • Dart plugin version 191.8593
    X Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[√] VS Code (version 1.42.1)
    • VS Code at C:\Users\Faruk Aydin\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.8.1

[√] Connected device (1 available)
    • Redmi 8A • f12007c00606 • android-arm • Android 9 (API 28)

! Doctor found issues in 2 categories.
exit code 0
```

**Comment by TahaTesser** [comment]
Hi @aydin-faruk 
Please refrain from posting log screenshots, Instead paste them here  in the comment. 
Can you please provide your `flutter run --verbose`?
Also, to better address the issue, would be helpful if you could post a minimal code sample to reproduce the problem
Thank you

**Comment by aydin-faruk** [comment]
@TahaTesser  I can not run "flutter run --verbose". 
I getting this message  "command not found"

**Comment by TahaTesser** [comment]
Hi @aydin-faruk 
Please [Update your path](https://flutter.dev/docs/get-started/install/windows#update-your-path) and then `flutter run --verbose` from your project
Also, to better address the issue, would be helpful if you could post a minimal code sample to reproduce the problem
Thank you

**Comment by aydin-faruk** [comment]
My path is last version. 
You can see in my question. I shared flutter doctor

**Comment by TahaTesser** [comment]
Hi @aydin-faruk 
I can see you have `Java 13`, please install `Java 8` [Solution](https://github.com/flutter/flutter/issues/47043#issuecomment-568781690)
Thank you

**Comment by aydin-faruk** [comment]
Hi My 
```
[  +13 ms] executing: [c:\flutter\] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +66 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] 0b8abb4724aa590dd0f429683339b1e045a1594d
[        ] executing: [c:\flutter\] git describe --match v*.*.* --first-parent --long --tags
[  +60 ms] Exit code 0 from: git describe --match v*.*.* --first-parent --long --tags
[        ] v1.12.13+hotfix.8-0-g0b8abb472
[   +9 ms] executing: [c:\flutter\] git rev-parse --abbrev-ref --symbolic @{u}
[  +65 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [c:\flutter\] git ls-remote --get-url origin
[  +51 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[ +115 ms] executing: [c:\flutter\] git rev-parse --abbrev-ref HEAD
[  +57 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[  +66 ms] "flutter run" took 121ms.
Error: No pubspec.yaml file found.
This command should be run from the root of your Flutter project.
Do not run this command from the root of your git clone of Flutter.

#0      FlutterCommand.validateCommand (package:flutter_tools/src/runner/flutter_command.dart:690:9)
#1      RunCommand.validateCommand (package:flutter_tools/src/commands/run.dart:285:19)
#2      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:590:11)
#3      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:515:33)
<asynchronous suspension>
#4      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:146:29)
#5      _rootRun (dart:async/zone.dart:1126:13)
#6      _CustomZone.run (dart:async/zone.dart:1023:19)
#7      _runZoned (dart:async/zone.dart:1518:10)
#8      runZoned (dart:async/zone.dart:1465:12)
#9      AppContext.run (package:flutter_tools/src/base/context.dart:145:18)
#10     FlutterCommand.run (package:flutter_tools/src/runner/flutter_command.dart:505:20)
#11     CommandRunner.runCommand (package:args/command_runner.dart:197:27)
#12     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:413:21)     
<asynchronous suspension>
#13     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:146:29)
#14     _rootRun (dart:async/zone.dart:1126:13)
#15     _CustomZone.run (dart:async/zone.dart:1023:19)
#16     _runZoned (dart:async/zone.dart:1518:10)
#17     runZoned (dart:async/zone.dart:1465:12)
#18     AppContext.run (package:flutter_tools/src/base/context.dart:145:18)
#19     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:19)
#20     CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:112:25)
#21     new Future.sync (dart:async/future.dart:224:31)
#22     CommandRunner.run (package:args/command_runner.dart:112:14)
#23     FlutterCommandRunner.run (package:flutter_tools/src/runner/flutter_command_runner.dart:250:18)
#24     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:63:22)
#25     _rootRun (dart:async/zone.dart:1126:13)
#26     _CustomZone.run (dart:async/zone.dart:1023:19)
#27     _runZoned (dart:async/zone.dart:1518:10)
#28     runZoned (dart:async/zone.dart:1502:12)
#29     run.<anonymous closure> (package:flutter_tools/runner.dart:61:18)
<asynchronous suspension>
#30     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:146:29)
#31     _rootRun (dart:async/zone.dart:1126:13)
#32     _CustomZone.run (dart:async/zone.dart:1023:19)
#33     _runZoned (dart:async/zone.dart:1518:10)
#34     runZoned (dart:async/zone.dart:1465:12)
#35     AppContext.run (package:flutter_tools/src/base/context.dart:145:18)
#36     runInContext (package:flutter_tools/src/context_runner.dart:64:24)
#37     run (package:flutter_tools/runner.dart:50:10)
#38     main (package:flutter_tools/executable.dart:65:9)
#39     main (file:///C:/b/s/w/ir/k/archive/flutter/packages/flutter_tools/bin/flutter_tools.dart:8:3)
#40     _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:305:32)
#41     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:174:12)
```

**Comment by aydin-faruk** [comment]
I deleted the JDK 13. I installed JDK 8.
I gave the environment variable for the Flutter SDK. in user variables
flutter
C: \ flutter \ bin

I gave the environment variable for JAVA_HOME. in system variables
JAVA_HOME
C: \ Program Files \ Java \ jdk1.8.0_241

Then I accepted Android licenses with cmd.
(flutter doctor --android-licenses)

Now I can use flutter connectivity.
The problem is solved. Thank you.
